scene/gui/file_dialog.cpp

```diff
@@ -187,6 +187,7 @@ void FileDialog::_notification(int p_what) {
 			}
 			refresh->set_icon(theme_cache.reload);
 			show_hidden->set_icon(theme_cache.toggle_hidden);
+			makedir->set_icon(theme_cache.create_folder);
```

Comment:
Is there a particular reason why makedir doesn't have color override just a bit after?

Reply:
Nicely done! Thanks!